### PR TITLE
[DEV-1580] foundation

### PR DESCRIPTION
**Pull request by Krysh90**

### CODE REVIEW

File: .github/workflows/prd.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+env:
+  NODE_VERSION: '16.x'
+  AZURE_ACCOUNT_NAME: 'stdfxlnxprd'
+  AZURE_CDN_PROFILE: 'cdnp-dfx-lnx-prd'
+  AZURE_CDN_ENDPOINT: 'cdne-dfx-lnx-prd'
+  AZURE_RESOURCE_GROUP: 'rg-dfx-lnx-prd'
+
+jobs:
+  build:
+    name: Build and deploy to DEV
```
Comment (davidleomay) [inline .github/workflows/prd.yml]: PRD

File: package.json [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+      },
+      {
+        "files": [
+          "**/*.stories.*"
+        ],
+        "rules": {
+          "import/no-anonymous-default-export": "off"
+        }
+      }
+    ]
```
Comment (davidleomay) [inline package.json]: Are there any stories?

File: src/validations.ts
Comment (davidleomay) [inline src/validations.ts:1]: Maybe this belongs to the react package (incl. blocked-iban.json)?

Comment (Krysh90) [reply src/validations.ts:1]: We can talk about it. First I have seen it as project based, but we could argue that it is the same for every project and therefore should belong to a package.

Comment (davidleomay) [reply src/validations.ts:1]: Added a ticket

**Review by davidleomay — APPROVED** [review]

**Pull request merged**